README.md

```diff
@@ -187,16 +188,15 @@ make test
 ## Running Benchmark
 
 ```bash
-go test -test.bench <pattern>
-go test -test.bench BenchmarkListRedis -benchmem
+make run_benchmark
 ```
 
 ## Getting Started
 
 To get started with building and contributing to DiceDB, please refer to the [issues](https://github.com/DiceDB/dice/issues) created in this repository.
 
 ## Docs
-
+<!-- TODO: Replace Netlify status badge with the appropriate API -->
```

Comment:
Shall we remove this?

Reply:
It was intentionally added to point out that netlify status badge is not working